test/unit/spec/util/sdp/index.js

```diff
@@ -490,6 +491,222 @@ a=ssrc:0000000000 label:d8b9a935-da54-4d21-a8de-522c87258244\r
+a=ssrc:0000000000 mslabel:7a9d401b-3cf6-4216-b260-78f93ba4c32e\r
+a=ssrc:0000000000 label:d8b9a935-da54-4d21-a8de-522c87258244\r
+a=ssrc:1111111111 cname:s9hDwDQNjISOxWtK\r
+a=ssrc:1111111111 msid:7a9d401b-3cf6-4216-b260-78f93ba4c32e d8b9a935-da54-4d21-a8de-522c87258244\r
+a=ssrc:1111111111 mslabel:7a9d401b-3cf6-4216-b260-78f93ba4c32e\r
+a=ssrc:1111111111 label:d8b9a935-da54-4d21-a8de-522c87258244\r
+`;
+    const remoteSdp = `\
+v=0\r
+o=- 6385359508499371184 3 IN IP4 127.0.0.1\r
+s=-\r
+t=0 0\r
+a=group:BUNDLE 0 1\r
+a=msid-semantic: WMS 7a9d401b-3cf6-4216-b260-78f93ba4c32e\r
+m=audio 22602 UDP/TLS/RTP/SAVPF 111 0\r
+c=IN IP4 34.203.250.135\r
+a=rtcp:9 IN IP4 0.0.0.0\r
+a=candidate:2235265311 1 udp 7935 34.203.250.135 22602 typ relay raddr 107.20.226.156 rport 51463 generation 0 network-cost 50\r
+a=ice-ufrag:Cmuk\r
+a=ice-pwd:qjHlb5sxe0bozbwpRSYqil3v\r
+a=ice-options:trickle\r
+a=fingerprint:sha-256 BE:29:0C:60:05:B6:6E:E6:EA:A8:28:D5:89:41:F9:5B:22:11:CD:26:01:98:E0:55:9D:FE:C2:F8:EA:4C:17:91\r
+a=setup:actpass\r
+a=mid:0\r
+a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level\r
+a=recvonly\r
+a=rtcp-mux\r
+a=rtpmap:111 opus/48000/2\r
+a=rtcp-fb:111 transport-cc\r
+a=fmtp:111 minptime=10;useinbandfec=1\r
+a=rtpmap:0 PCMU/8000\r
+m=video 9 UDP/TLS/RTP/SAVPF 102\r
+c=IN IP4 0.0.0.0\r
+a=rtcp:9 IN IP4 0.0.0.0\r
+a=ice-ufrag:Cmuk\r
+a=ice-pwd:qjHlb5sxe0bozbwpRSYqil3v\r
+a=ice-options:trickle\r
+a=fingerprint:sha-256 BE:29:0C:60:05:B6:6E:E6:EA:A8:28:D5:89:41:F9:5B:22:11:CD:26:01:98:E0:55:9D:FE:C2:F8:EA:4C:17:91\r
+a=setup:actpass\r
+a=mid:1\r
+a=extmap:2 urn:ietf:params:rtp-hdrext:toffset\r
+a=extmap:3 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time\r
+a=extmap:4 urn:3gpp:video-orientation\r
+a=extmap:5 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01\r
+a=extmap:6 http://www.webrtc.org/experiments/rtp-hdrext/playout-delay\r
+a=extmap:7 http://www.webrtc.org/experiments/rtp-hdrext/video-content-type\r
+a=extmap:8 http://www.webrtc.org/experiments/rtp-hdrext/video-timing\r
+a=recvonly\r
+a=rtcp-mux\r
+a=rtcp-rsize\r
+a=rtpmap:102 H264/90000\r
+a=rtcp-fb:102 goog-remb\r
+a=rtcp-fb:102 transport-cc\r
+a=rtcp-fb:102 ccm fir\r
+a=rtcp-fb:102 nack\r
+a=rtcp-fb:102 nack pli\r
+a=fmtp:102 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42e01f\r
+a=ssrc-group:FID 0000000000 1111111111\r
+a=ssrc:0000000000 cname:s9hDwDQNjISOxWtK\r
+a=ssrc:0000000000 msid:7a9d401b-3cf6-4216-b260-78f93ba4c32e d8b9a935-da54-4d21-a8de-522c87258244\r
+a=ssrc:0000000000 mslabel:7a9d401b-3cf6-4216-b260-78f93ba4c32e\r
+a=ssrc:0000000000 label:d8b9a935-da54-4d21-a8de-522c87258244\r
+a=ssrc:1111111111 cname:s9hDwDQNjISOxWtK\r
+a=ssrc:1111111111 msid:7a9d401b-3cf6-4216-b260-78f93ba4c32e d8b9a935-da54-4d21-a8de-522c87258244\r
+a=ssrc:1111111111 mslabel:7a9d401b-3cf6-4216-b260-78f93ba4c32e\r
+a=ssrc:1111111111 label:d8b9a935-da54-4d21-a8de-522c87258244\r
```

Comment:
it would be useful to see the resultant/updated sdp as part of this test.

Reply:
@makarandp0 We generally don't log intermediate results.